src/frontend/packages/lib_video/src/components/live/common/InputDisplayNameOverlay/index.spec.tsx

```diff
@@ -47,6 +53,10 @@ const mockSetLiveSessionDisplayName =
     typeof setLiveSessionDisplayName
   >;
 
+const live = liveMockFactory({
+  id: 'some-live-id',
```

Comment:
I don't think it is needed to set the `id`.

Reply:
It helps to mock the URL as it is not interesting to have a dynamic UUID here.